manifests/2.4.0/opensearch-2.4.0.yml

```diff
@@ -92,6 +92,14 @@ components:
     checks:
       - gradle:properties:version
       - gradle:dependencies:opensearch.version
+  - name: neural-search
+    repository: https://github.com/opensearch-project/neural-search.git
+    ref: '2.x'
+    platforms:
+      - linux
```

Comment:
Can you please add `-windows` as well for neural search?
Thanks

Reply:
> neural search needs to be added after ml-commons and k-NN because it takes these as dependencies.

Given k-NN is not added for Windows, and k-NN is deendency of neural-search, I skipped adding for k-NN.

Reply:
Got it. Thanks.